core/src/main/java/org/mskcc/cbio/portal/model/GeneticAlterationType.java

```diff
@@ -53,5 +53,6 @@
     PROTEIN_ARRAY_PROTEIN_LEVEL,
     PROTEIN_ARRAY_PHOSPHORYLATION,
     GENESET_SCORE,
+    TREATMENT,
```

Comment:
1. For profiles the name TREATMENT should be renamed to TREATMENT_RESPONSE. This to distinguish it from the TREATMENT genetic entity.
2. Why is the reference to GENERIC_ASSAY still here. It is obsolete not?